test/src/test/java/org/corfudb/infrastructure/logreplication/LogReplicationFSMTest.java

```diff
@@ -66,9 +67,9 @@ public class LogReplicationFSMTest extends AbstractViewTest implements Observer
     private static final String TEST_STREAM_NAME = "StreamA";
     private static final int BATCH_SIZE = 2;
     private static final int WAIT_TIME = 100;
-    private static final int CORFU_PORT = 9000;
     private static final int TEST_TOPOLOGY_CONFIG_ID = 1;
     private static final String TEST_LOCAL_CLUSTER_ID = "local_cluster";
+    private static final String TEST_LOCAL_ENDPOINT_PREFIX = "test:";
```

Comment:
correct thing would be to put this in AbstractViewTest and replace "test:" in code, so if it changes in one place all take effect.

Reply:
It might come from dependency issue, but this test class is not at the same package as `AbstractViewTest`. I tend to defer addressing this as other test files also need similar changes.